### PR TITLE
configure.ac: detect target, if target is darwin (OSX), then

### PR DESCRIPTION
**Pull request by mooredan**
Builds and operates on OSX Yosemite using MacPorts osxfuse

### CODE REVIEW

File: src/s3fs.cpp [hunk 1]
```diff
@@ -2748,6 +2749,7 @@ static void* s3fs_init(struct fuse_conn_info* conn)
   if(is_remove_cache && !FdManager::DeleteCacheDirectory()){
     DPRNINFO("Could not inilialize cache directory.");
   }
+  #endif
```
Comment (ggtakec) [inline src/s3fs.cpp]: For __APPLE__ case, does FdManager::DeleteCacheDirectory() occur something error?

**Comment by mooredan** [comment]
Oops, that #endif is in the wrong spot, it should be a few lines up. I'll correct this shortly.  Good catch!

**Comment by mooredan** [comment]
Fixed pull request

**Comment by ggtakec** [comment]
@mooredan Thanks for fixing. I merged this.

**Pull request merged**